dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventChangeLog.java

```diff
@@ -40,6 +41,6 @@ public record Change(@JsonProperty DataValueChange dataValue) {}
 
   public record DataValueChange(
       @JsonProperty String dataElement,
-      @JsonProperty String previousValue,
-      @JsonProperty String currentValue) {}
+      @JsonInclude @JsonProperty String previousValue,
```

Comment:
Why is `@JsonInclude` needed in addition to `@JsonProperty`?

Reply:
to see json elements with null values, if not they are skipped

Reply:
This is different to what we usually do 🤔 Did we decide on whether we would add a `type` field for `create`, `update`, `delete`?

Reply:
it's not required by design, front end didn't answer me yet on whether they'd like to see it while testing.

Reply:
We decided not return the keys if the values are null as its consistent with what we do.